packages/live-status-gateway/src/topics/adLibsTopic.ts

```diff
@@ -32,6 +32,7 @@ export interface AdLibsStatus {
 	rundownPlaylistId: string | null
 	adLibs: AdLibStatus[]
 	globalAdLibs: GlobalAdLibStatus[]
+	templateAdLibs?: TemplateAdLibStatus[]
```

Comment:
Are `templateAdLibs` only global? Or are they always Segment- or Part-local? I think it would be good to make that clear. Or should they still exists on individual `adLibs` and `globalAdLibs` topics and have a `template` or `optionsRequired` flag turned on?

Reply:
templateAdLibs are no longer a separate array. They can now be discovered by finding a template property in the public data.